### PR TITLE
Server-side PCD verification

### PR DESCRIPTION
**Pull request by cedoor**

### CODE REVIEW

File: apps/client/src/App.tsx [hunk 1]
```diff
@@ -1,75 +1,78 @@
...
-
-                    setBgColor(color)
-                } else {
-                    alert(`The signature is not valid!`)
+                if (!response.ok) {
+                    alert("Some error occurred")
+                    return
                 }
+
+                alert(`The signature is valid, the background color will be changed`)
```
Comment (0xjei) [inline apps/client/src/App.tsx]: nit: We can improve this `alert` to deal with the following edge cases

- Starting background color: `white`. Consume PCD with a signed message equal to `white`. Background color from `white` to `white` (no UI differences).
- Starting background color: `white`. Consume PCD with a signed message equal to `red`. Background color from `white` to `red` (UI okay). Then, consume another different PCD with a signed message equal to `red`. Background color from `red` to `red` (no UI differences).

**Review by 0xjei — APPROVED** [review]
Just a small nit but LGTM. Feel free to make the change or merge as-is :)

File: apps/client/src/App.tsx (hunk 1 above)
Comment (cedoor) [reply apps/client/src/App.tsx]: Right, I can check whether the current background color is the same and if so, show another message instead. How does it sound?

Comment (0xjei) [reply apps/client/src/App.tsx]: Definitely better for the demo purposes!

**Pull request merged**